kratos/solving_strategies/schemes/residualbased_incrementalupdate_static_scheme.h

```diff
@@ -10,113 +10,82 @@
-
-\URL[Example of use pdf]{ extended_documentation/no_ex_of_use.pdf}
-
-\URL[Example of use doc]{ extended_documentation/no_ex_of_use.doc}
-
-\URL[Example of use ps]{ extended_documentation/no_ex_of_use.ps}
-
-
-\URL[Extended documentation html]{ extended_documentation/no_ext_doc.html}
-
-\URL[Extended documentation pdf]{ extended_documentation/no_ext_doc.pdf}
-
-\URL[Extended documentation doc]{ extended_documentation/no_ext_doc.doc}
-
-\URL[Extended documentation ps]{ extended_documentation/no_ext_doc.ps}
-
-
-*/
+///@name Kratos Globals
+///@{
+///@}
+///@name Type Definitions
+///@{
+///@}
+///@name  Enum's
+///@{
+///@}
+///@name  Functions
+///@{
+///@}
+///@name Kratos Classes
+///@{
+
+/** 
+ * @class ResidualBasedIncrementalUpdateStaticScheme
+ * @ingroup KratosCore
+ * @brief This class provides the implementation of a static scheme
+ * @details The only operation done in this  scheme is the update of the database, no predict is done
+ * @tparam TSparseSpace The sparse space considered
+ * @tparam TDenseSpace The dense space considered
+ * @see Scheme
+ * @author Riccardo Rossi
+ */
 template<class TSparseSpace,
          class TDenseSpace //= DenseSpace<double>
          >
 class ResidualBasedIncrementalUpdateStaticScheme : public Scheme<TSparseSpace,TDenseSpace>
 {
 
 public:
-    /**@name Type Definitions */
-    /*@{ */
-
+    ///@name Type Definitions
+    ///@{
+    
+    /// Pointer definition of ResidualBasedIncrementalUpdateStaticScheme
     KRATOS_CLASS_POINTER_DEFINITION( ResidualBasedIncrementalUpdateStaticScheme);
 
-    typedef Scheme<TSparseSpace,TDenseSpace> BaseType;
-
-    typedef typename BaseType::TDataType TDataType;
-
-    typedef typename BaseType::DofsArrayType DofsArrayType;
-
-    typedef typename BaseType::TSystemMatrixType TSystemMatrixType;
+    /// Base class definition
+    typedef Scheme<TSparseSpace,TDenseSpace>                      BaseType;
 
-    typedef typename BaseType::TSystemVectorType TSystemVectorType;
+    /// DoF array type definition
+    typedef typename BaseType::DofsArrayType                 DofsArrayType;
 
+    /// Data type definition
+    typedef typename BaseType::TDataType                         TDataType;
+    /// Matrix type definition
+    typedef typename BaseType::TSystemMatrixType         TSystemMatrixType;
+    /// Vector type definition
+    typedef typename BaseType::TSystemVectorType         TSystemVectorType;
+    /// Local system matrix type definition
     typedef typename BaseType::LocalSystemVectorType LocalSystemVectorType;
-
+    /// Local system vector type definition
     typedef typename BaseType::LocalSystemMatrixType LocalSystemMatrixType;
 
+    /// Elements containers definition
     typedef ModelPart::ElementsContainerType             ElementsArrayType;
-
+    /// Conditions containers definition
     typedef ModelPart::ConditionsContainerType         ConditionsArrayType;
 
-
-    /*@} */
-    /**@name Life Cycle
-    */
-    /*@{ */
+    ///@}
```

Comment:
is this parsed correctly by doxygen? in the sense is the message contained in the middle correctly grouped together?

Reply:
In my IDE, Kdevelop, I can group the doxygen, in fact it generates the doc for the current file on the air, so I suppose this is correct